<launch>

    <node pkg="mitre_fast_layered_map" type="static_map_node" name="static_map" output="screen">
        <rosparam command="load" file="$(find mitre_fast_layered_map)/config/static_map.yaml"/>
    </node>

</launch>